ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1039,3 +1039,18 @@ literals:
 - fivesecondrolling_rootmeansquare_zaxis_linearacceleration_sensor
 - local_air_temperature_sensor
 
+# Added on 2021/09/22 for lighting control
+- zone_lighting_switch_status:
+  - ON
+  - OFF
+- zone_lighting_switch_command:
+  - ON
+  - OFF
+- zone_brightness_percentage_command
+- lighting_switch_alarm:
+  - ACTIVE
+  - INACTIVE
+
+# Added on 2021/09/22 for water leak
+- water_cable_extent_flow_sensor
```

Comment:
I need this explained. By the sound of it you are trying to combine both flow and length in some way. Please describe what this device does and what this field in particular measures.

Reply:
yes sir. the sensor is a long cable that returns distance in meter where it touches water. The cable is laid on the floor to surround the room (so that it returns distance in meters where in the cable is the water detected). 

the best i can come up with is "water_leak_cable_extent_level_sensor" (or maybe "water_leak_cable_sensor"). level seems to imply vertical, i need something that is horizontal linear length (unfortunately there is no "linear" or "length" in measurement subfield).  something like "water_leak_cable_perimeter_sensor" or "water_leak_cable_length_sensor" (aside from length maybe distance or perimeter or linear or i dunno).